save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityInfoTab.kt

```diff
@@ -103,22 +106,36 @@ val vulnerabilityInfoTab: FC<VulnerabilityInfoTabProps> = FC { props ->
     }
 
     @Suppress("TYPE_ALIAS", "MAGIC_NUMBER", "UnusedPrivateProperty")
-    val projectTable: FC<TableProps<VulnerabilityProjectDto>> = useMemo(i18n.language) {
+    val projectTable: FC<CosvTableProps> = useMemo(i18n.language) {
         tableComponent(
             columns = {
                 columns {
-                    column(id = "name", header = "Name".t(), { name }) { cellContext ->
+                    column(id = "name", header = "Name".t(), { `package`?.name }) { cellContext ->
```

Comment:
can it be null actually? if so

`package`?.name ?: "????"

Reply:
Fixed